README.md

```diff
@@ -27,19 +53,20 @@ The data model/DB schema lives in `j-db-client/prisma/schema`, server side code
    $ git clone git@github.com:Journaly/journaly.git
    ```
 1. Locate the `.env.example` file and copy the contents into a new `.env` file alongside it.
+1. Navigate to `packages/web`
 1. Run `npm ci`
-1. Run `npm run prisma:generate`
 
 #### Setting Up Your Local DB Instance
 
 1. Install Postgres
+
 - Mac/Linux users should use Homebrew
```

Comment:
Linux users should _probably_ be using their distro's postgres package manager rather than linux homebrew